PodcastGenerator/admin/navbar.php

```diff
@@ -1,12 +1,13 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-danger">
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+<div class="container"> 
     <a class="navbar-brand" href="index.php">Admin</a>
     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="<?php echo _('Toggle navigation'); ?>">
         <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarNav">
         <ul class="navbar-nav">
             <li class="nav-item">
-                <a class="nav-link" href="index.php">Home</a>
+                <a class="nav-link" href="../index.php">Home</a>
```

Comment:
Why are we changing the path of index.php to the parent directory?

Reply:
Right...
I have nested the navbar inside a container (looks a bit more clear) making this, the navbar is less wider and i couldnt fit all the previous menu links inside it...
Since the "Admin" and "Home" in the administrator navbar were pointing to the same link i have removed "View Podcast" which pointed to the main homepage, and modified "Home" so it will point to the main page (like the "view podcast" button did previously).

Hope i made it clear :)